content/ember/v4/link-to-disabled-when.md

```diff
@@ -0,0 +1,8 @@
+---
+id: ember.link-to.disabled-when
+title: Passing disabledWhen to LinkTo
+until: '5.0.0'
+since: '4.0.0'
```

Comment:
What should these versions be? See PR https://github.com/emberjs/ember.js/pull/19748 and https://github.com/emberjs/ember.js/pull/19748